nep-27.mediawiki

```diff
@@ -0,0 +1,42 @@
+NEP: 27
```

Comment:
Likely should be 28. Ref. https://github.com/neo-project/proposals/pull/169#discussion_r1608980280.